go.mod

```diff
@@ -52,7 +52,7 @@ require (
 	code.cloudfoundry.org/rep v0.0.0-20200325195957-1404b978e31e // indirect
 	code.cloudfoundry.org/rfc5424 v0.0.0-20180905210152-236a6d29298a // indirect
 	code.cloudfoundry.org/tlsconfig v0.0.0-20200131000646-bbe0f8da39b3 // indirect
-	github.com/DataDog/agent-payload v4.42.0+incompatible
+	github.com/DataDog/agent-payload v4.42.1-0.20200825141519-fca03e52c077+incompatible
```

Comment:
will be fixed after this is merged:  https://github.com/DataDog/agent-payload/pull/65